docs/src/chapters/dalsze-plany-rozwoju.adoc

```diff
@@ -1 +1,48 @@
 == Dalsze plany rozwoju
+
+W ramach pierwszego wydania nie zostały ukończone wszystkie początkowo zaplanowane zadania. Jest to związane z nierównym
+wkładem władnym członków zespołu projektowego. W rejestrze zadań znajduje się jeszcze kilkanaście zadań. Większość
+z nich dotyczy zmniejszenia długu technicznego oraz mniej znaczących poprawek. Znalazło się tam również ważne
+zadania. Funkcjonalność odtwarzania portfela dodana jest tylko po stronie serwera. Integracja procesu w aplikacji
+internetowej nie została dodana. Drugą ważną funkcjonalnością jest dynamiczne obliczanie opłaty transakcyjnej w
+zależności od rozmiaru transakcji. W aktualnej wersji jest to sztywno ustalona kwota, która powinna być zaakceptowana
+w większości przypadków. Opłata transakcyjna powinna być obliczana automatycznie na podstawie danych o opłatach z
+ostatnich bloków przeliczonych przez rozmiar bloku w bajtach. W aplikacji zabrakło również kompletnych testów E2E.
+Zrealizowano je tylko dla procesu uwierzytelniania, a w planach były jeszcze dla całej reszty aplikacji internetowej
+oraz aplikacji mobilnej. Planowane też było usprawnienie procesu budowania produktów dostarczanych. Na potrzeby
+pierwszego wydania pliki te zostały zbudowane ręcznie, natomiast w rejestrze znajdują się zadania automatyzujące
+ten proces w ramach procesu CI/CD. Ostatnią brakującą funkcjonalnością jest przesyłanie zakupionych tokenów na adres
+email użytkownika. Już w karcie wymagań projektu zadanie było oznaczone jako opcjonalne, więc odpowiedni wpis nie
+znalazł się w rejestrze. Planowano jego realizację tylko w przypadku ukończenia projektu długo przed terminem, co nie
+miało miejsca. Wszystkie z wymienionych nieukończonych funkcjonalności będą ukończone w najbliższym czasie i zostaną
+uwzględnione w następnym wydaniu.
+
+Aplikacja ma duży potencjał do rozwoju. Nowe funkcjonalności można dodawać praktycznie bez końca. Projekt powstał
+jako połączenie dwóch innych projektów, które miały powstać w ramach projektu _Ingenious_. Był to serwer do przyjmowania
+płatności wraz z generowaniem tokenów oraz portfel multisig z aplikacją mobilną do podpisywania transakcji.
+Naturalną rzeczą jest, że połączenie dwóch projektów skutkowało zmniejszeniem liczby funkcjonalności obu z nich.
+Wiele z nich planowane jest na przyszłe wydania. Moduł płatności powinien być bardziej uniwersalny. Jedną z możliwości
+jest dodanie funkcjonalności pozwalającej na dostosowanie stylu aplikacji przez administratorów. Może to się odbywać
+przykładowo poprzez kaskadowe arkusze stylów CSS. Dobrym pomysłem dyskutowanym na początku było stworzenie serwera z
```

Comment:
CSS to znaczy kaskadowe arkusze stylów
więc albo to albo tamto

Reply:
done